Cargo.toml

```diff
@@ -31,11 +31,13 @@ name = "ggez"
 path = "src/lib.rs"
 
 [features]
-default = ["c_dependencies"]
+default = ["c_dependencies", "audio", "gamepad"]
 bzip2 = ["zip/bzip2"]
 mp3 = ["rodio/mp3"]
 multithread-image-decoding = ["image/hdr", "image/jpeg_rayon"]
 c_dependencies = ["bzip2", "mp3"]
```

Comment:
Currently opting out of rodio by disabling "audio" would still fail, as "mp3" still links to rodio and is part of the default features through "c_dependencies", leading to rodio being loaded and compiled anyway.

Reply:
What's `c_dependencies` for?

Reply:
> Currently opting out of rodio by disabling "audio" would still fail

Also, this is not possible; You can't disable a single feature. You need to disable all default features to enable one specifically.

Reply:
`c_dependencies` groups together all libraries that at some point depend on a C-compiler, which can be undesirable, and luckily what they provide is all non-essential so they may be disabled.

You’re right, it’s a silly thought. Who would disable default features and then reenable `c_dependencies` without `audio`? And if you only need `bzip` or `mp3` but not the other there’s already features for that so it’s fine.